db-server/test/backend/db_tests.js

```diff
@@ -334,7 +334,7 @@ module.exports = function (config, DB) {
         return db.updateSessionToken(sessionTokenData.tokenId, sessionTokenUpdates)
           .then((result) => {
             assert.deepEqual(result, {}, 'Returned an empty object on session token update')
-            return db.sessionToken(sessionTokenData.tokenId)
+            return db.sessionWithDevice(sessionTokenData.tokenId)
```

Comment:
Updated corresponding tests to use `sessionWithDevice`. This meant that I had to also add checks for `tokenVerificationId` since that value is returned.